docs/src/api/class-browsertype.md

```diff
@@ -75,6 +75,7 @@ This methods attaches Playwright to an existing browser instance.
 * langs: js
 - `params` <[Object]>
   - `wsEndpoint` <[string]> A browser websocket endpoint to connect to.
+  - `extraHTTPHeaders` <[Object]<[string], [string]>> Additional HTTP headers to be sent with web socket connect request. Optional.
```

Comment:
these are not really 'extra', just `httpHeaders` or `headers`

Reply:
done

Reply:
There are other headers as well but if think it's ok to user shorter name `headers`. Renamed.